{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2017-18265"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "Prosody before 0.10.0 allows remote attackers to cause a denial of service (application crash), related to an incompatibility with certain versions of the LuaSocket library, such as the lua-socket package from Debian stretch. The attacker needs to trigger a stream error. A crash can be observed in, for example, the c2s module."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "NVD-CWE-noinfo"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://bugs.debian.org/875829",
					"tags": [
						"Issue Tracking",
						"Mailing List"
					],
					"url": "https://bugs.debian.org/875829"
				},
				{
					"name": "https://bugs.debian.org/875829",
					"tags": [
						"Issue Tracking",
						"Mailing List"
					],
					"url": "https://bugs.debian.org/875829"
				},
				{
					"name": "https://hg.prosody.im/0.9/rev/176b7f4e4ac9",
					"tags": [
						"Patch"
					],
					"url": "https://hg.prosody.im/0.9/rev/176b7f4e4ac9"
				},
				{
					"name": "https://hg.prosody.im/0.9/rev/176b7f4e4ac9",
					"tags": [
						"Patch"
					],
					"url": "https://hg.prosody.im/0.9/rev/176b7f4e4ac9"
				},
				{
					"name": "https://hg.prosody.im/0.9/rev/adfffc5b4e2a",
					"tags": [
						"Patch"
					],
					"url": "https://hg.prosody.im/0.9/rev/adfffc5b4e2a"
				},
				{
					"name": "https://hg.prosody.im/0.9/rev/adfffc5b4e2a",
					"tags": [
						"Patch"
					],
					"url": "https://hg.prosody.im/0.9/rev/adfffc5b4e2a"
				},
				{
					"name": "https://prosody.im/issues/issue/987",
					"tags": [
						"Vendor Advisory"
					],
					"url": "https://prosody.im/issues/issue/987"
				},
				{
					"name": "https://prosody.im/issues/issue/987",
					"tags": [
						"Vendor Advisory"
					],
					"url": "https://prosody.im/issues/issue/987"
				},
				{
					"name": "DSA-4198",
					"tags": [
						"Technical Description"
					],
					"url": "https://www.debian.org/security/2018/dsa-4198"
				},
				{
					"name": "DSA-4198",
					"tags": [
						"Technical Description"
					],
					"url": "https://www.debian.org/security/2018/dsa-4198"
				}
			]
		}
	},
	"impact": {
		"baseMetricV2": {
			"cvssV2": {
				"accessComplexity": "LOW",
				"accessVector": "NETWORK",
				"authentication": "NONE",
				"availabilityImpact": "PARTIAL",
				"baseScore": 5,
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"vectorString": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
				"version": "2.0"
			},
			"exploitabilityScore": 10,
			"impactScore": 2.9,
			"severity": "MEDIUM"
		},
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "HIGH",
				"baseScore": 7.5,
				"baseSeverity": "HIGH",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"privilegesRequired": "NONE",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
				"version": "3.0"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 3.6
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:prosody:prosody:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "0.10.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			},
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T03:19Z",
	"publishedDate": "2018-05-09T17:29Z"
}
